meteor/server/coreSystem.ts

```diff
@@ -539,7 +539,7 @@ function startInstrumenting() {
 			transactionSampleRate: system.apm.transactionSampleRate,
 			disableMeteorInstrumentations: ['methods', 'http-out', 'session', 'async', 'metrics'],
 		})
-		profiler.setActive(true)
+		profiler.setActive(system.apm.enabled || true)
```

Comment:
Shouldnt this be `|| false`?
`undefined || true` = true
`false || true` = true
`true || true` = true